src/styles/typography/link-on-dark-background/index.njk

```diff
@@ -2,7 +2,7 @@
 title: Links on dark backgrounds – Typography
 layout: layout-example.njk
 stylesheets:
-- ../inverse.css
+  - ../../../stylesheets/example-inverse.css
```

Comment:
I think this has been Prettier-formatted (recommended YAML indent)

Not a blocker, but all the other `stylesheets:` array items aren't indented